schema.sql

```diff
@@ -1,10 +1,12 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    id int NOT NULL IDENTITY(1,1),
+    id              INT,
     name            CHAR(20),
     date_of_birth   DATE,
     escape_attempts INT, 
     neutered        BOOLEAN,
     weight_kg       DECIMAL
 );
+
+ALTER TABLE animals ADD species CHAR(20);
```

Comment:
- [ ] I think you can use the VARCHAR data type which has a variable length limit rather than CHAR with fixed length